package.json

```diff
@@ -2,18 +2,24 @@
   "name": "mo54",
   "version": "0.1.1",
   "scripts": {
-    "build": "node ./Scripts/devEnv.js HTML JS CSS"
+    "build": "node ./Scripts/devEnv.js HTML JS CSS AS",
+    "test": "asc -v "
   },
   "dependencies": {
     "as-wasi": "^0.4.6",
-    "assemblyscript": "^0.19.9"
+    "assemblyscript": "^0.20.6"
   },
   "devDependencies": {
     "@babel/cli": "^7.15.7",
     "@babel/core": "^7.15.5",
     "@babel/preset-env": "^7.15.6",
     "@babel/runtime": "^7.15.4",
-    "jshint": "^2.13.4",
+    "@html-eslint/eslint-plugin": "^0.13.1",
+    "@html-eslint/parser": "^0.13.1",
+    "@typescript-eslint/eslint-plugin": "^5.22.0",
+    "@typescript-eslint/parser": "^5.22.0",
+    "eslint": "^8.15.0",
+    "typescript": "^4.6.4",
     "uglify-js": "^3.14.2"
```

Comment:
uglify-js : 3.15.4 ?

Reply:
it's a minification lib it will be replaced with a more portent one. it is also out of the scope of this PR :)